examples/sinatra/views/profile.erb

```diff
@@ -1,62 +1,136 @@
-      <td><img src="selfie.jpeg" /></td>
-    </tr>
-    <tr>
-      <td>Full name</td>
-      <td><%= full_name %></td>
-    </tr>
-    <tr>
-      <td>Given names</td>
-      <td><%= given_names %></td>
-    </tr>
-    <tr>
-      <td>Family name</td>
-      <td><%= family_name %></td>
-    </tr>
-    <tr>
-      <td>Mobile number</td>
-      <td><%= mobile_number %></td>
-    </tr>
-    <tr>
-      <td>Email address</td>
-      <td><%= email_address %></td>
-    </tr>
-    <tr>
-      <td>Date of birth</td>
-      <td><%= date_of_birth %></td>
-    </tr>
-    <tr>
-      <td>Age verified</td>
-      <td><%= age_verified %></td>
-    </tr>
-    <tr>
-      <td>Address</td>
-      <td><%= address %></td>
-    </tr>
-    <tr>
-      <td>Gender</td>
-      <td><%= gender %></td>
-    </tr>
-    <tr>
-      <td>Nationality</td>
-      <td><%= nationality %></td>
-    </tr>
-  </table>
+<%
+attributes = [
+  {
+    "label"=> "Given names",
+    "obj"=> profile.given_names,
+    "icon"=> "yoti-icon-profile"
+  },
+  {
+    "label"=> "Family names",
+    "obj"=> profile.family_name,
+    "icon"=> "yoti-icon-profile"
+  },
+  {
+    "label"=> "Mobile number",
+    "obj"=> profile.phone_number,
+    "icon"=> "yoti-icon-phone"
+  },
+  {
+    "label"=> "Email address",
+    "obj"=> profile.email_address,
+    "icon"=> "yoti-icon-email"
+  },
+  {
+    "label"=> "Date of birth",
+    "obj"=> profile.date_of_birth,
+    "icon"=> "yoti-icon-calendar"
+  },
+  {
+    "label"=> "Age verified",
+    "label_prefix"=> "Age Verification/",
+    "obj"=> profile.age_condition,
+    "icon"=> "yoti-icon-verified"
+  },
+  {
+    "label"=> "Address",
+    "obj"=> profile.postal_address,
+    "icon"=> "yoti-icon-address"
+  },
+  {
+    "label"=> "Gender",
+    "obj"=> profile.gender,
+    "icon"=> "yoti-icon-gender"
+  }
+]
+%>
+<!DOCTYPE html>
+<html class="yoti-html">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <title>Yoti client example</title>
+    <link href="/css/profile.css" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Roboto:400,700" rel="stylesheet">
+  </head>
+  <body class="yoti-body">
+  <main class="yoti-profile-layout">
+      <section class="yoti-profile-user-section">
+        <div class="yoti-profile-picture-powered-section">
+          <span class="yoti-profile-picture-powered">Powered by</span>
+          <a href="https://www.yoti.com/" target="_blank">
+              <img class="yoti-logo-image" src="/assets/logo.png" srcset="/assets/logo@2x.png 2x" alt="Yoti" />
+          </a>
+        </div>
+
+        <div class="yoti-profile-picture-section">
+          <div class="yoti-profile-picture-area">
+            <img src="selfie.jpeg" class="yoti-profile-picture-image" alt="Yoti" />
+            <i class="yoti-profile-picture-verified-icon"></i>
+          </div>
+
+          <% if profile.full_name %>
```

Comment:
Do you think we should list this in the `attributes` section starting on line 2, so it's clearer that this is an attribute too?

Reply:
I think it's clear that it's an attribute as it's getting a profile method, but it might not clearly visible. For that reason I can set it at the top level.